x-pack/test/functional_with_es_ssl/config.base.ts

```diff
@@ -126,6 +126,14 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
               spaces: ['*'],
             },
           ],
+          elasticsearch: {
```

Comment:
What is the reason for adding this?

Reply:
See https://github.com/elastic/kibana/pull/193948#issuecomment-2388301800

Reply:
But before it was working without it, why does this role (`alerts_and_actions_role`) need extra permission? Did some tests start failing because we added the `rac` API privilege?

Reply:
But before it was working without it, why does this role (`alerts_and_actions_role`) need extra permission? Did some tests start failing because we added the `rac` API privilege?

Reply:
I just added it to simulate a correct scenario (currently the user needs access to the index to be able to see the controls bar) but if we don't mind seeing the error toast in the functional test I can remove it, the other tests work just fine

Reply:
I think is fine what I do not get is why the `alerts_and_actions_role` role needs it and not the `stackAlertsOnlyReadSpacesAll` which is used by the tests. I am sure I am missing something 🙂.